### PR TITLE
MGDAPI-4548 fix the URLs and marin3r pod status check

### PR DESCRIPTION
**Pull request by MStokluska**
# Issue link
https://issues.redhat.com/browse/MGDAPI-4548
https://issues.redhat.com/browse/MGDAPI-4592

# What
- update to move all sops to gitlab instead of github repo
- small update to the logic around confirming marin3r pods state

# Verification steps
- Install RHOAM 1.26.0 via OLM
- Upgrade to RHOAM 1.27.0 using this branch as the source of the container image for 1.27.0
- Navigate to Observe > alerting in OSD and confirm that RHOAM alerts point to gitlab instead of github
- Navigate to Prometheus (you can get the route from observability namepspace) and go through each alert and confirm that they all have been moved to GitLab and no more github alerts are present.
- Logic/code check on rate limit pods

### CODE REVIEW

**Comment by carlkyrillos** [comment]
Reviewed changed files and completed the verification steps. Confirmed that all alerts that were pointing to `github.com/RHCloudServices/integreatly-help` are now pointing to `gitlab.cee.redhat.com/rhcloudservices/integreatly-help`. Also verified that the update to the logic around confirming the marin3r pod's state makes sense and that the rateLimitService unit test is still passing.
/lgtm

File: pkg/resources/sop_url.go [hunk 1]
```diff
@@ -2,51 +2,55 @@ package resources
...
+	SopUrlThreeScaleDeveloperUIBBT                             = "https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleDeveloperUIBBT.asciidoc"
+	SopUrlThreeScaleSystemAdminUIBBT                           = "https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/blob/master/sops/rhoam/alerts/ThreeScaleSystemAdminUIBBT.asciidoc"
+	SopUrlPodDistributionIncorrect                             = "https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/blob/master/sops/multi-az/pod_distribution.md"
+	SopUrlSloRhssoAvailabilityAlert                            = "https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/blob/master/sops/rhoam/alerts/SloRhssoAvailabilityAlert.asciidoc"
+	SopUrlSloUserSsoAvailabilityAlert                          = "https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/blob/master/sops/rhoam/alerts/SloUserSsoAvailabilityAlert.asciidoc"
+	SopUrlTestFireAlerts                                       = "https://gitlab.cee.redhat.com/rhcloudservices/integreatly-help/blob/master/sops/rhoam/cssre_info/info_test_fire_alerts.md#resolve-test-alerts"
```
Comment (austincunningham) [inline pkg/resources/sop_url.go:45]: this one seems to be dead

Comment (austincunningham) [reply pkg/resources/sop_url.go:45]: viewing issues on gitlab. all good now

**Comment by MStokluska** [comment]
bypassing the codecov

**Pull request merged**